frontends/ol-components/src/components/NavDrawer/NavDrawer.tsx

```diff
@@ -127,10 +128,11 @@ export interface NavItem {
   icon?: string | ReactElement
   description?: string
   href: string
+  onClick?: () => void
```

Comment:
This doesn't seem to be used? Let's remove it to avoid expanding the interface.

Reply:
It's newly used ([L191](https://github.com/mitodl/mit-learn/pull/1739/files#diff-2959b2acb45249092d040d1b51c53e9b4d47ff54e29b6f588972d8d6b5593c06R191)) to pass the drawer close when the link is clicked.